src/test/java/io/github/spark_redshift_community/spark/redshift/InMemoryS3AFileSystem.java

```diff
@@ -108,6 +110,8 @@ public FSDataInputStream open(Path f, int bufferSize) throws IOException {
 
     @Override
     public FSDataOutputStream create(Path f) throws IOException {
+        // TODO: This could possibly be wrong
+        FileSystem.Statistics statistics = null;
```

Comment:
feels possibly wrong to me too.  Without looking, maybe should be `new FileSystem.Statistics()`? Or whatever the constructor api is

Reply:
addressed.